src/plugins/intel_gna/src/gna_device.hpp

```diff
@@ -94,14 +94,14 @@ class GNADeviceHelper : public ov::intel_gna::GNADevice {
     void releaseModel(const uint32_t model_id);
     static uint32_t getNumberOfGnaDevices();
     static uint32_t selectGnaDevice();
-    static bool isGnaHw(const Gna2DeviceVersion dev) {
-        return Gna2DeviceVersionSoftwareEmulation != dev;
+    static bool is_hw_target(const common::DeviceVersion device_version) {
+        return common::DeviceVersion::SoftwareEmulation != device_version;
     }
-    bool hasGnaHw() const {
-        return isGnaHw(detectedGnaDevVersion);
+    bool is_hw_detected() const {
+        return is_hw_target(target->get_detected_device_version());
```

Comment:
hmm interesting that we require both methods to have public is_hw_target and is_hw_detected (this is static)
do we need this static method at all?

Reply:
is_up_to_20_hw will be made private.